src/components/Badge/Badge.tsx

```diff
@@ -0,0 +1,24 @@
+import * as classNames from 'classnames';
+import * as React from 'react';
+import { rivetize } from '../Rivet';
+
+interface BadgeProps {
+    secondary?: boolean,
```

Comment:
Is this `secondary` prop being used anywhere? I personally prefer this API versus having to specify when something should be the default.

Reply:
I refactored the `secondary` property from a previous iteration to be called `role` instead and had missed removing this property.  I also prefer this style of property, but I went with the convention for now.  If we want to tweak that convention I would certainly agree with it.